### PR TITLE
add example for timingwheel

### PR DESCRIPTION
**Pull request by tsingson**
add example

### CODE REVIEW

**Comment by RussellLuo** [comment]
@tsingson Thanks for your contribution! The example code you provide is correct and well-formatted, but I think it's a little complicated.

While using the same library (here is `timingwheel`), different users have different use-cases and different opinions about how to structure their code. So I think the examples of `timingwheel` should try to avoid:

- involving use-case specific things
    + in your example code, like periodically printing out the current time from `timingWheelTaskRunner()`, or continuously printing out the current time in `for i := 1; i <= MainGoroutineWait; i++ ` loop.
- or showing two many structuring styles
    + in your example code, like defining three global variables `tw`, `taskChannel` and `tl`, and initializing them within `init()`, or defining many small functions such as `timingwheelTaskTriggerFunc()`, `timingWheelTaskRunner()`,  `goInternalTimerTaskRunner()` and `goInternalTimer()`. 

In other words, being a basic library, `timingwheel` should provide examples **as concentrated (also simple) as possible**, just like what most standard Go libraries do.

So if I understand your example code correctly, the core snippet that is concentrated on how to use `timingwheel` is mostly the same with the first example (i.e. `Example (AddTimer)`) provided in [Godoc](https://godoc.org/github.com/RussellLuo/timingwheel)?

**Comment by RussellLuo** [comment]
The API to create a new timer has been changed (by this [commit](https://github.com/RussellLuo/timingwheel/commit/f7d111032da78650784c3a10b77cc76a7c289e8e)), see [Godoc](https://godoc.org/github.com/RussellLuo/timingwheel) for new examples.

I hope the new API will be considered more user-friendly.

**Comment by tsingson** [comment]
Thank you for your quick response.

Yes, my example is a bit complicated and not easy to understand.
Perhaps this is related to personal coding habits.
